config/zones/CA-ON.yaml

```diff
@@ -87,6 +87,11 @@ emissionFactors:
       - datetime: '2021-01-01'
         source: Electricity Maps, 2021 average
         value: 57.47323263502428
+    gas:
+      datetime: '2008-01-01'
+      source: Mallia, E., Lewis, G. "Life cycle greenhouse gas emissions of electricity \n
+        generation in the province of Ontario, Canada." (2013)
```

Comment:
Why are there newlines in the source? This also don't match the source provided in the top level sources object.